infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -40,47 +43,62 @@
  * In this scheme, the key for each value is also the name of the file where the value is stored.
  * The key is determined as (prefix + "_" + key).
  * The cache here serves mostly for easily managed synchronization of in-memory/file.
+ * Some on-disk files will be deleted when their number exceed user specified limit.
  *
  * <p>If 'opts' either has '--memory=true' or a log-path for storing files is not provided,
  * the store is just an in memory cache.
  *
  * <p>Created by mdhawan on 7/27/16.
  */
 
+@Slf4j
 public class DataStore implements IDataStore {
 
     static String EXTENSION = ".ds";
 
     @Getter
     private final Cache<String, Object> cache;
-    private final String logDir;
+    private final String logDirPath;
```

Comment:
streamLogDirPath

Reply:
this the data logs path, stream log is different.